version/version.go

```diff
@@ -17,4 +17,6 @@ package version
 var (
 	// Version of Kogito Operator
 	Version = "7.11.0"
+	// PAMVersion used for metering labels
+	PAMVersion = "7.10"
```

Comment:
Why 7.10 and not 7.11? :thinking:

Reply:
I was following @ibek's comment in [the Jira](https://issues.redhat.com/browse/RHPAM-3681?focusedCommentId=16364445&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16364445).

Reply:
I think it should be 7.12.0 at this point. or, 7.11.1;

Reply:
The version updates are done in `bump-version.sh` so I think we can put 7.11.0 here, will be updated before release to proper version together with other places.

Reply:
I don't think we need to add this env as well, component version should be the same than the pro version.